api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -152,7 +156,107 @@ public void saveUser_shouldShouldCreateUserWhoIsPatientAlready() throws SQLExcep
 		List<Patient> allPatientsSet = Context.getPatientService().getAllPatients();
 		assertEquals(1, allPatientsSet.size());
 	}
-	
+
+	@Test(expected = APIException.class)
```

Comment:
@wisogetnid can you please use the `expectedException` junit rule as done in other tests in this class